.github/workflows/build-images.yaml

```diff
@@ -192,48 +192,29 @@ jobs:
           else
           echo "VERSION=${{ github.sha }}" >> $GITHUB_ENV
           fi
-      - name: Install qemu dependency
-        run: |
-          sudo apt-get update
-          sudo apt-get install -y qemu-user-static
-      - name: Run make catalog (main)
-        if: ${{ github.ref_name == env.MAIN_BRANCH_NAME }}
-        run: |
-          make catalog \
```

Comment:
This make target has been recently refactored to replace the deprecated Sqlite-based generation of the catalog with file-based builds (#201).

Also, I'd rather stay with single way of doing it between local dev env and CI, if possible. So I wish there's a way for us to keep relying on the make target here.

Reply:
`opm index add` generate Operator index container images from pre-existing Operator bundles and builds as well. Since it does both the jobs so thats why we have removed `make catalog` options

Reply:
I'm not sure if we can rollback to using `opm index add` for generating the catalog. I'm gonna have to ask people smarter than me on this one, I'm afraid.

cc @didierofrivia @eguzki

Reply:
if that means going back to sqlite based catalogs, I'd say it's not a good idea.

Reply:
@didierofrivia `opm index add` doesn't mean it will fall back to sqlite based catalogs. Below snapshot has been taken from OpenShift Docs. It says that default Red Hat provided Operator Catalog releases in the file based catalog format from 4.11 and we have used 4.14.4 version of OPM
![WhatsApp Image 2024-10-03 at 20 29 40_172a3e58](https://github.com/user-attachments/assets/e0295079-7f3c-4bb6-950a-1eeb6f2b9e52)

Reply:
For what it's worth, inspecting one of the pushed images (for example quay.io/r3hankhan/authorino-operator-catalog:latest-s390x), I'm pretty sure they're sqlite based and not file-based catalogs (looking at the last layer)